Makefile

```diff
@@ -3,6 +3,8 @@
 IMG ?= quay.io/fpaoline/metallboperator:latest
```

Comment:
can u we reset the above to controller too ?

Reply:
I'm not sure about that, keeping it like this causes make deploy (without any additional args) to deploy the operator with the latest image. I feel that if we change it to controller people will use make deploy, it won't work, and then they'll need to navigate Makefile/documentation to understand what's wrong.

Reply:
no I am using make deploy for the controller and it bringup the operator with the right image , give it a try it all driven by IMG env var once its set all will just work

Reply:
got it u wanted to default image to CI build